src/main/java/net/minecraftforge/items/IItemHandler.java

```diff
@@ -85,6 +88,19 @@
      */
     int getSlotLimit(int slot);
```

Comment:
Should this method be deprecated and eventually phased out? Passing `ItemStack.EMPTY` to `getMaxStackSize` can be defined (and imo should be defined) to return the same as `getSlotLimit`.

Reply:
I think this one will need some more discussion. I get the reasoning for not having two versions of the method, but I'd like to know if people are generally okay with the semantics of empty stack = generic context.

Reply:
Should ItemStack.EMPTY be considered a stackable item in that case?